### PR TITLE
Update styleguide.config.js

### PR DESCRIPTION
**Pull request by andrasbacsai**
Add disableHostCheck to work in codesandbox.

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ module.exports = {
 	title: 'React Style Guide Example',
 	defaultExample: true,
 	webpackConfig: {
+		devServer: {
+			disableHostCheck: process.env.CODESANDBOX_SSE ? true : false,
```
Comment (sapegin) [inline styleguide.config.js]:
```suggestion
			disableHostCheck: !!process.env.CODESANDBOX_SSE,
```

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks! I have one suggestion. Also: is there any documentation we can link in a comment about this environmental variable any why we need this code?

**Comment by andrasbacsai** [comment]
Yep [here](https://codesandbox.io/docs/environment#how-can-i-tell-if-i-am-in-a-container-sandbox) it is.

**Review by sapegin — APPROVED** [review]
👍

**Pull request merged**